pkg/brokerapi/openservicebroker/open_service_broker_client.go

```diff
@@ -40,9 +40,10 @@ const (
 	serviceInstanceAsyncFormatString       = "%s/v2/service_instances/%s?accepts_incomplete=true"
 	serviceInstanceDeleteFormatString      = "%s/v2/service_instances/%s?service_id=%s&plan_id=%s"
 	serviceInstanceDeleteAsyncFormatString = "%s/v2/service_instances/%s?service_id=%s&plan_id=%s&accepts_incomplete=true"
-	pollingFormatString                    = "%s/v2/service_instances/%s/last_operation"
+	pollingFormatString                    = "%s/v2/service_instances/%s/last_operation?%s"
```

Comment:
What I meant to say is: as per spec, they _should_ be sent and since we control the code for sending them, they will be always sent in our controller.